test-one/src/index.js

```diff
@@ -1,14 +1,24 @@
 import { log } from "./helpers/logger";
 
 export const go = async () => {
-  // Your code should go here instead of the welcome message below. Create and modify additional files as necessary/appropriate.
+    var counter;
+    var displayMessage = "";
 
-  const welcomeMessage =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! " +
-    "Please replace this message with your solution and feel free to add " +
-    "or modify other files as necessary. Good luck : )";
+    for (counter = 1 ; counter <= 100; counter++) {
```

Comment:
Are you aware of a plugin/library that could help keep code style consistent? No biggie, but I can see use of semi-colon and whitespace inconsistencies here.

Reply:
I normally code in visual studio to keep code styling consistent, although I must of went back and changed one of the values which interfered with the styling of that part.